src/main/pages/che-7/administration-guide/con_kubernetes-image-puller-overview.adoc

```diff
@@ -0,0 +1,24 @@
+[id="kubernetes-image-puller-overview_{context}"]
+= Kubernetes Image Puller overview
+
+Slow starts of {prod} workspaces may be caused by the necessity to pull container images from remote registries. Pre-pulling images can improve start times significantly. The _Kubernetes Image Puller_ can be used to pre-pull images and shorten workspace start times.
+
+The Kubernetes Image Puller is an additional deployment that runs alongside {prod}. Given a list of images to pre-pull, the application runs inside a cluster and creates a _daemonset_ that pulls the images on each node.
+
+The application can be deployed via Helm or by processing and applying OpenShift templates.
+
+The Kubernetes Image Puller pulls its configuration from a `ConfigMap` with the following available parameters:
+
+[id="kubernetes-image-puller-configuration"]
+.Kubernetes Image Puller parameters
+[options="header"]
+|===
+|Parameter |Usage |Default
+|`CACHING_INTERVAL_HOURS` |Interval, in hours, between checking health of daemonsets |`"1"`
+|`CACHING_MEMORY_REQUEST` |The memory request for each cached image when the puller is running |`10Mi`
+|`CACHING_MEMORY_LIMIT` |The memory limit for each cached image when the puller is running |`20Mi`
```

Comment:
I'd include in this section why a user might want to set a non-default value of memory limit/request, or maybe what effect it has on the deployment.

Reply:
@tomgeorge, this is the last suggestion to address before we can merge:

> I'd include in this section why a user might want to set a non-default value of memory limit/request, or maybe what effect it has on the deployment.